storage/interface.go

```diff
@@ -154,6 +154,8 @@ type SeriesSet interface {
 	Next() bool
 	At() Series
 	Err() error
+	// A collection of warnings for the whole set.
```

Comment:
Let's to the same for Err. 

Also let's mention that it also is there for both error and non error sets